enumStruct/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace enumStruct
+{
+    class Program
+    {
+        //Bunch of loosely related stuff in a struct.
+        public struct Monster
+        {
+            public string name;
+            public int health;
+            public int damage;
+            public Element element;
+            //Example type comparison
+            public static bool compareType(Element e1, Element e2)
+            {
+                if (e1 == Element.Fire)
+                {
+                    if (e2 == Element.Water)
+                    {
+                        return "Fire is weak against water.";
+                    }
```

Comment:
Nice, for extra bonus points, how would you create an app, where the code is the same to play RockPaperScissors, but i get to choose what the "characters" are (ie, rock/paper/scissors, wate/fire/nature, cockroach/boot/nuclearBomb") etc ...